bundles/org.openhab.automation.luascripting/README.md

```diff
@@ -0,0 +1,3 @@
+# Lua Scripting
+
+This add-on provides support for [Lua](https://www.lua.org/) 5.2 scripts in openHAB.
```

Comment:
Can you add some more details to the readme similar to what we've added for Jython/Groovy recently?